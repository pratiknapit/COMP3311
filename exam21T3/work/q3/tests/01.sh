psql property -c 'select * from q3;'
