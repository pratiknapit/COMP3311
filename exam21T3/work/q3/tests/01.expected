  id   |  price  |       street        |  suburb   
-------+---------+---------------------+-----------
 46083 | 1742000 | 86 Leonard Avenue   | Kingsford
 46634 | 1742000 | 88 Broadbent Street | Kingsford
(2 rows)
